tests/unit-core/test-snapshot.c

```diff
@@ -223,7 +223,7 @@ main (void)
     /* Check the snapshot data. Unused bytes should be filled with zeroes */
     const uint8_t expected_data[] =
     {
-      0x4A, 0x52, 0x52, 0x59, 0x22, 0x00, 0x00, 0x00,
+      0x4A, 0x52, 0x52, 0x59, 0x23, 0x00, 0x00, 0x00,
```

Comment:
What has changed if this feature does not work with snapshots?

Reply:
New cbc opcodes has been added.